instagram/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import './App.css';
+import SearchBar from './SearchBar/SearchBar';
+import PostContainer from './PostContainer/PostContainer';
+import dummyData from './dummy-data';
+import PropTypes from "prop-types";
+
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state = {
+      data: [],
+    }
+  }
+
+  componentDidMount() {
+    this.setState({
+      data: dummyData
+    })
+  }
+
+  render() {
+    return (
+      <div className="App">
+          <SearchBar data={this.state.data}/>
+          {this.state.data.map(item =>{
+            return(
+            <PostContainer obj={item} key={item.timestamp} />
```

Comment:
Try to use naming that is more specific to what is being passed down in props to the children components, such as userData, instead of passing obj